config/checkstyle/checkstyle.xml

```diff
@@ -11,15 +11,15 @@ android checkstyle
 -->
 <module name="Checker">
   <module name="SuppressWarningsFilter"/>
-  <module name="SuppressionCommentFilter" />
   <module name="SuppressionFilter">
     <property name="file" value="${checkstyleConfigDir}/suppressions.xml"/>
   </module>
   <property name="severity" value="warning"/>
   <module name="TreeWalker">
-    <module name="FileContentsHolder"/>
     <module name="SuppressWarningsHolder" />
     <property name="tabWidth" value="4"/>
+    <module name="SuppressionCommentFilter" />
+<!--
```

Comment:
May I ask why are we commenting these out?

Reply:
nvm.. just noticed this is a draft pr.

Reply:
Because they don't pass right now.  My goal is to have a checkstyle that passes and fails the build if it fails.  Then work on removing commented rules.